tests/gold_tests/pluginTest/multiplexer/multiplexer.test.py

```diff
@@ -96,33 +106,37 @@ def setupTS(self, skip_post):
                 "proxy.config.ssl.client.verify.server.policy": 'PERMISSIVE',
                 'proxy.config.diags.debug.enabled': 1,
                 'proxy.config.diags.debug.tags': 'http|multiplexer',
+                'proxy.config.dns.nameservers': f'127.0.0.1:{self.dns.Variables.Port}',
+                'proxy.config.dns.resolv_conf': 'NULL',
             })
         self.ts.Disk.ssl_multicert_config.AddLine('dest_ip=* ssl_cert_name=server.pem ssl_key_name=server.key')
         skip_remap_param = ''
         if skip_post:
             skip_remap_param = ' @pparam=proxy.config.multiplexer.skip_post_put=1'
         self.ts.Disk.remap_config.AddLines(
             [
-                f'map https://origin.server.com https://127.0.0.1:{self.server_origin.Variables.https_port} '
+                f'map https://origin.server.com https://backend.origin.server.com:{self.server_origin.Variables.https_port} '
```

Comment:
Does MicroDNS return the loopback address for any hostname it wasn't configured with?  If so, does that mean we can't use MicroDNS to test DNS lookup failures?

Reply:
Typically, we configure the default to resolve everything to 127.0.0.1. In this PR, you can see that here:

https://github.com/apache/trafficserver/pull/10965/files#diff-ab6abc922014fa32a8c9d9a16e3b9be6e89f16a45a671728146d270ba2d848cfR49

I _think_ MicroDNS can be used to test failures, but I'm not sure offhand. It might require some modification of it.